aws/resource_aws_api_gateway_integration_test.go

```diff
@@ -272,11 +272,10 @@ func TestAccAWSAPIGatewayIntegration_integrationType(t *testing.T) {
 				),
 			},
 			{
-				Config: testAccAWSAPIGatewayIntegrationConfig_IntegrationTypeInternet(rName),
```

Comment:
This test is the same as [Line 259](https://github.com/terraform-providers/terraform-provider-aws/pull/15499/files#diff-b1909efafe91a419f2290615c3c0e80bR259) so instead of creating another test I've replaced the duplicate with the new test

Reply:
If I recall correctly, the update back to `INTERNET` was an intentional update test. In general, the maintainers prefer if existing testing is not touched to ensure regressions are covered.